libs/community/langchain_community/vectorstores/azuresearch.py

```diff
@@ -740,6 +739,18 @@ class AzureSearchVectorStoreRetriever(BaseRetriever):
     or "semantic_hybrid_score_threshold"."""
     k: int = 4
     """Number of documents to return."""
+    search_kwargs: dict = Field(
+        default_factory=lambda: {"fetch_k": 20, "lambda_mult": 0.5}
```

Comment:
can we just default to empty dict here and let vecstore set defaults?

Reply:
@baskaryan done